mxnet/inference/docker/1.7.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,181 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="3"
+
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.6.10
+ARG MX_URL=https://gluon-pypi.s3-us-west-2.amazonaws.com/1.7.0/gluon-1.7.0-py2.py3-none-manylinux2014_x86_64.whl
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=0.9.2
```

Comment:
The smdebug binary is not usually included in the inference container.

Reply:
Done